pvr.iptvsimple/resources/language/resource.language.en_gb/strings.po

```diff
@@ -259,7 +259,12 @@ msgctxt "#30067"
 msgid "Use FFMpeg http reconnect options if possible"
 msgstr ""
 
-#empty strings from id 30068 to 30099
+#. label: Advanced - userAgent
+msgctxt "#30068"
+msgid "User-Agent"
```

Comment:
Maybe `User agent to use if not supplied`

Reply:
I would like to reserve some space at the end of the line for writing.

Reply:
Good point makes sense 👍